scanpipe/tests/data/basic-rootfs_root_filesystems.json

```diff
@@ -31,24 +31,26 @@
             "distro": {
               "os": "linux",
               "architecture": null,
-              "name": "Debian GNU/Linux",
-              "version": "9 (stretch)",
-              "identifier": "debian",
-              "id_like": null,
-              "version_codename": null,
-              "version_id": "9",
-              "pretty_name": "Debian GNU/Linux 9 (stretch)",
+              "name": "Ubuntu",
```

Comment:
Here I've updated the `os-release` file in the basic-rootfs test case to the ubuntu distro file, as the packages detected below seem to be ubuntu packages, and the old example seemed to be wrong. Otherwise the namespaces being overridden from this `os-release` file seemed to be wrong.